docs/source/usage.rst

```diff
@@ -13,3 +13,17 @@ This package is useful for obtaining important financial information about publi
 - Business Profile
 - Letter to Shareholders
 - Management's Analysis
+
+The ``Filing`` class provides a simple API to fetch SEC filings.
+
+.. ipython:: python
+
+   from SECEdgar.filings import Filing
+   my_cik = '0000320193'
+   my_filings = Filing(cik=my_cik, filing_type='10-q', count=15)
```

Comment:
Filing type can be an `Enum` rather than a string. https://docs.python.org/3/library/enum.html. It will help in not passing an ill-formatted string.

```
class FilingType(Enum):
       TENQ = "10-Q"

```

Reply:
So this would change the code to look something more like

```python
my_filings = Filing(cik=my_cik, filing_type=FilingType.TENQ, count=15)
```

My only concern is that "TENQ" doesn't seem as clear as something like "10Q" (although this name is not possible).

A few ways to change this would be to change the enum names (ex:  `_10Q = "10-Q"`) or to only provide one accepted spelling for the `filing_type` (ex: while `'10-q'` and `'10q'` are both valid now, we could change so that only `'10-q'` is valid)